src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function (a, b) {
+      return a + b;
```

Comment:
[CODE KNOWLEDGE] - When defining methods inside an object, you should use the shorthand method syntax without the `function` keyword. For example, instead of `add: function (a, b) { return a + b; }`, you should write `add(a, b) { return a + b; }`.